other-docs/getting-started/differences-from-wp.md

```diff
@@ -0,0 +1,86 @@
+
+You'll also see a `vendor/` directory, as well as some extra files including `composer.json` and `.build-script`.
+
+
+## Dependencies and versioning
+
+With Altis projects, you use [Composer](https://getcomposer.org/) to manage dependencies including Altis and WordPress. You can also [manage plugins and themes via Composer](third-party-plugins.md) if you want.
+
+Your `composer.json` contains your Composer manifest. This describes which libraries and tools to load in. If you've just created a new version of Altis, this will only contain Altis itself (`altis/altis`) as well as the local environments (`altis/local-chassis` and `altis/local-server`). These will be tied to a specific version of Altis (like `^10.0.0`).
+
+The main Altis package (`altis/altis`) will load in a bunch of dependencies, including WordPress and bundled plugins. **The version of WordPress you use is tied to the Altis version**, as are any bundled plugins. This means we can ensure all the functionality works well together. We support each major version [for a full year](docs://guides/long-term-support/) including backporting bugfixes and security patches.
+
+As the developer, you're responsible for keeping this version up to date. We recommend using [Dependabot](https://github.blog/2020-06-01-keep-all-your-packages-up-to-date-with-dependabot/) for automated alerts about new major and minor versions of Altis.
+
+
+## Installing plugins and themes
+
+Unlike many WordPress hosts, on Altis your codebase is managed through GitHub. This means any changes to your project's code need to go through version control.
+
+In our cloud environments, we operate a read-only filesystem for security reasons. This means that **users cannot install or upgrade plugins or themes**, and changes must go via the codebase instead.
+
+You can [manage plugins and themes via Composer](third-party-plugins.md), which can integrate with Dependabot. Alternatively, you can use a local environment to check the Upgrades tab inside the WordPress dashboard, then commit any changes to your repository.
+
+
+## Ecosystem compatibility
+
+Altis is broadly compatible with the WordPress ecosystem, including most WordPress plugins and themes. Our [automated code review](docs://guides/code-review/) checks for known security and performance issues.
+
+We have a few changes from standard WordPress. The biggest one of those is that the filesystem is read-only. That means instead of having a `wp-content/uploads/` directory, we use [S3 for storage](docs://cloud/s3-storage/). We automatically set up filters internally which mean most plugins work, but some plugins which need filesystem access may break.
+
+Notably, most backup plugins will not work on Altis. Altis operates [automated backup processes for you](docs://cloud/backups/), and the Altis Dashboard allows you to take a manual snapshot of your uploads or database if you need a copy (such as for testing locally).
```

Comment:
" Altis operates [automated backup processes for you]" should be 
" Altis operates an [automated backup processes for you]"

Reply:
"processes" though, not "process"